components/Procedure/src/stories/bem.stories.mdx

```diff
@@ -29,82 +29,85 @@ import "../index.scss";
 <Canvas>
   {/* eslint-disable react/no-unknown-property */}
   <Story name="Default">
-    <ol class="denhaag-process-steps denhaag-procedure">
-      <li class="denhaag-process-steps__step denhaag-process-steps__step--current">
-        <div class="denhaag-process-steps__step-header">
-          <div class="denhaag-step-marker denhaag-step-marker--current"></div>
-          <p class="denhaag-process-steps__step-heading utrecht-heading-4">Maak een afspraak</p>
-        </div>
-        <div class="denhaag-process-steps__step-body">
-          <div class="denhaag-step-marker__connector denhaag-step-marker__connector--current denhaag-step-marker__connector--main-to-nested"></div>
-          <div class="denhaag-process-steps__step-meta">
-            <p class="utrecht-paragraph">
-              Voor het aanvragen van een nieuw paspoort moet u langskomen op Stadhuis Spui of stadskantoor Leyweg. U
-              maakt daarvoor een afspraak.
-            </p>
+    <div class="denhaag-procedure-container">
+      <h3 class="denhaag-procedure__heading">Title</h3>
```

Comment:
<div class="denhaag-procedure-container"> + <h3 class="denhaag-procedure__heading">Title</h3> toegevoegd.

De rest is Prettier formatting.